velox/dwio/dwrf/common/IntEncoder.h

```diff
@@ -369,4 +387,26 @@ void IntEncoder<isSigned>::writeLongLE(int64_t val) {
   }
 }
 
+template <bool isSigned>
+void IntEncoder<isSigned>::writeVuHugeInt(uint128_t val) {
+  while (true) {
+    if ((val & ~0x7f) == 0) {
+      writeByte(static_cast<char>(val));
+      return;
+    } else {
+      writeByte(static_cast<char>(0x80 | (val & dwio::common::BASE_128_MASK)));
+      // Cast val to unsigned so as to force 0-fill right shift.
+      val >>= 7;
+    }
+  }
+}
+
+template <bool isSigned>
+void IntEncoder<isSigned>::writeHugeIntLE(int128_t val) {
+  for (int32_t i = 0; i < numBytes_; i++) {
```

Comment:
We should use auto i = 0 or uint32_t i = 0 because numBytes_ type is uint32_t, I would prefer `auto`